File: med-visit-service/src/main/java/com/otbs/medVisit/dto/MedicalVisitResponse.java
package com.otbs.medVisit.dto;

import java.time.LocalDate;
import java.time.LocalTime;

public record MedicalVisitResponse (
    Long id,
    String doctorName,
    LocalDate visitDate,
    LocalTime startTime,
    LocalTime endTime,
    Integer numberOfAppointments
)
{}

File: app/src/main/java/com/example/onetechbs/db/MedicalVisitResponse.kt
package com.example.onetechbs.db

import com.google.gson.annotations.SerializedName

data class MedicalVisitResponse(
    val id: Long,
    @SerializedName("doctorName")
    val doctorName: String,
    @SerializedName("visitDate")
    val visitDate: String,
    @SerializedName("startTime")
    val startTime: String,
    @SerializedName("endTime")
    val endTime: String
)
